tests/02-checkrun.sh

```diff
@@ -29,3 +29,6 @@ check_run_ok() {
 #Check wrong combinations
 check_run_ok "ansible-deploy run -t task_exec_bin_true -s prod -i testInfra"
 
+check_output_fail "ansible-deploy run -t task_exec_bin_ERR -s prod -i testInfra"
```

Comment:
check_output_fail should be used with two args - second is error message to look for. Nice addition would be to check in check_output_fail if rc wasn't 0. (take a look at 01-...sh